torch/_inductor/scheduler.py

```diff
@@ -577,8 +577,9 @@ def get_buf_bytes(buf: Optional[Union[ir.Buffer, ir.TensorBox]]) -> int:
                     users = self.scheduler.name_to_buf[buf.get_name()].users
                     tot = 0
                     for user in users:
-                        assert isinstance(user.node, BaseSchedulerNode)
-                        if isinstance(user.node.node, MultiOutput):
+                        if isinstance(user.node, BaseSchedulerNode) and isinstance(
+                            user.node.node, MultiOutput
+                        ):
```

Comment:
@Chillee wonder would this change be okay? we now have `OutputNode` instead of `BaseSchedulerNode` as a potential user node, and I believe to follow the spirit of the existing code, this case should fall into the else branch below and return 0, so making this change here. cc. @bdhirsh

Reply:
nw - I fixed the issue and this doesn't appear anymore